File: shell-ui/src/test/java/dev/sayaya/handbook/client/redirect/Application.java
package dev.sayaya.handbook.client.redirect;

import com.google.gwt.core.client.EntryPoint;
import dev.sayaya.handbook.client.api.FetchMock;
import elemental2.dom.DomGlobal;
import elemental2.dom.HTMLButtonElement;
import jsinterop.base.Js;
import jsinterop.base.JsPropertyMap;

public class Application implements EntryPoint {
    private final Component components = DaggerComponent.create();
    @Override
    public void onModuleLoad() {
        components.historyManager().initialize();
        components.workspaceRepository().list().subscribe(list -> {});
    }
}

File: gateway/src/main/kotlin/dev/sayaya/handbook/Application.kt
package dev.sayaya.handbook

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

/** API 게이트웨이 Spring Boot 애플리케이션 진입점. 라우팅, 메뉴 집계, 서비스 디스커버리를 담당한다. */
@SpringBootApplication
class Application

fun main(args: Array<String>) {
    runApplication<Application>(*args)
}